helpers/exceptions.js

```diff
@@ -18,6 +18,10 @@ module.exports = {
 	rounds: {
 		'27040': {rewards_factor: 2, fees_factor: 2, fees_bonus: 10000000}
 	},
+	blockRewards: [
```

Comment:
yup these exceptions are only present in testnet branches.

We could declare the array empty though, right?